src/main/kotlin/util/MinecraftTemplates.kt

```diff
@@ -23,7 +23,7 @@ class MinecraftTemplates : FileTemplateGroupDescriptorFactory {
 
         FileTemplateGroupDescriptor("Bukkit", PlatformAssets.BUKKIT_ICON).let { bukkitGroup ->
             group.addTemplate(bukkitGroup)
-            bukkitGroup.addTemplate(FileTemplateDescriptor(BUKKIT_MAIN_CLASS_TEMPLATE))
+            bukkitGroup.addTemplate(FileTemplateDescriptor(BUKKIT_JAVA_MAIN_CLASS_TEMPLATE))
```

Comment:
I don't see the Kotlin variant added anywhere.

As I mentioned in another comment it might be better to have a single base template name and register templates with all tge language variants supported.
We could have a `FileTemplateGroupDescriptor.addTemplate(name: String, vararg languages: LanguageType)` extension method simply appending the base `name` with the file languages extension. Or a `template` overload with the same arguments and logic (see the bottom of this file)